src/init/features/apphosting/index.ts

```diff
@@ -132,6 +160,56 @@ export async function doSetup(
   logSuccess(`Your site is now deployed at:\n\thttps://${backend.uri}`);
 }
 
+/**
+ * Prompts the user for the web app that they would like to associate their backend with
+ * @param projectId user's projectId
+ * @param existingUserProjectWebApps a map of a user's web apps to their ids
+ * @return the name and ID of a web app
+ */
+async function promptFirebaseWebAppId(
```

Comment:
We should create a web app if there are none, are you planning on adding that in another PR?

Reply:
not that's implemented here, see: https://github.com/firebase/firebase-tools/pull/6875/files#diff-1f7254b41cdcf666adebb8a934648480cbb16a8abffe6145254f564aed2ec315R198

Reply:
I actually just implemented that functionality: https://github.com/firebase/firebase-tools/pull/6875/files#diff-1f7254b41cdcf666adebb8a934648480cbb16a8abffe6145254f564aed2ec315R168